cmd/cone/task_search.go

```diff
@@ -120,3 +120,21 @@ func (r *TaskSearchResponse) Rows() [][]string {
 
 	return rows
 }
+
+func (r *TaskSearchResponse) WideHeader() []string {
+	task := Task{}
+	return task.WideHeader()
+}
+
+func (r *TaskSearchResponse) WideRows() [][]string {
+	rows := [][]string{}
+	for _, task := range r.List {
+		t := Task{
+			task:   task.Task,
+			client: nil,
```

Comment:
if client is nil this will panic if `pretext` is called. Probably worth just adding the client to the `TaskSearchResponse`

I want to overall rethink this pattern though